cedar-policy-cli/tests/sample.rs

```diff
@@ -1021,3 +1021,37 @@ fn test_require_policies_for_write() {
             "the following required arguments were not provided:\n  --policies <FILE>",
         ));
 }
+
+#[test]
+fn test_json_policy() {
+    let json_policies: &str = "sample-data/tiny_sandboxes/json_policy/policy.cedar.json";
+    let entities: &str = "sample-data/tiny_sandboxes/json_policy/entity.json";
+
+    assert_cmd::Command::cargo_bin("cedar")
+        .expect("bin exists")
+        .arg("check-parse")
+        .arg("--policy-format")
+        .arg("json")
+        .arg("-p")
+        .arg(json_policies)
+        .assert()
+        .code(0);
+
+    assert_cmd::Command::cargo_bin("cedar")
```

Comment:
Do these other tests in this file check allow/deny behavior? If so, you might want to add this. If not, then nevermind 🙂

Reply:
Exit code `0` implies allow